bedrock/mozorg/redirects.py

```diff
@@ -16,8 +16,6 @@ def decider(request, **kwargs):
 redirectpatterns = (
     # bug 755826, 1222348, 1416798
     redirect(r"^zh-CN/?$", "https://www.firefox.com.cn/", locale_prefix=False, query={"utm_medium": "referral", "utm_source": "mozilla.org"}),
-    # bug 874913, 681572
-    redirect(r"^(products/)?download\.html", "firefox.new", query=""),
```

Comment:
I think this is unrelated to your redirect issues and you can / should restore this.